src/burst_db/build_frame_db.py

```diff
@@ -0,0 +1,743 @@
+        con.execute(
+            "UPDATE gpkg_geometry_columns SET geometry_type_name = 'MULTIPOLYGON';"
+        )
+        # Set the relative_orbit_number as the most common value for each frame
+        con.execute("ALTER TABLE frames ADD COLUMN relative_orbit_number INTEGER;")
+        con.execute(
+            """WITH frame_tracks AS (
+                SELECT f.fid,
+                    CAST(ROUND(AVG(b.relative_orbit_number)) AS INTEGER) AS relative_orbit_number
+                FROM frames f
+                JOIN frames_bursts fb ON f.fid = fb.frame_fid
+                JOIN burst_id_map b ON fb.burst_ogc_fid = b.ogc_fid
+                GROUP BY 1
+            ) UPDATE frames SET relative_orbit_number = frame_tracks.relative_orbit_number
+            FROM frame_tracks
+            WHERE frames.fid = frame_tracks.fid;
+            """
+        )
+        # Set the orbit_pass as the value from the first burst
+        con.execute("ALTER TABLE frames ADD COLUMN orbit_pass TEXT;")
+        con.execute(
+            """WITH op AS
+                (SELECT f.fid,
+                        b.orbit_pass
+                FROM frames f
+                JOIN frames_bursts fb ON f.fid = fb.frame_fid
+                JOIN burst_id_map b ON fb.burst_ogc_fid = b.ogc_fid),
+            frame_orbits AS (
+                SELECT fid,
+                    FIRST_VALUE(orbit_pass) OVER (PARTITION BY fid) AS orbit_pass
+                FROM op
+                GROUP BY fid)
+            UPDATE frames SET orbit_pass = frame_orbits.orbit_pass
+            FROM frame_orbits
+            WHERE frames.fid = frame_orbits.fid;
+            """
+        )
+
+        # Drop the is_land from the frames_bursts table
+        con.execute("ALTER TABLE frames_bursts DROP COLUMN is_land;")
+
+
+def get_epsg_codes(df: gpd.GeoDataFrame):
+    """Get the EPSG codes for all non-antimeridian polygons in a GeoDataFrame.
+
+    Uses the UTM library to account for the oddities of the Zones near Norway [1]_.
+
+    References
+    ----------
+    .. _[1]: http://www.jaworski.ca/utmzones.htm
+    """
+
+    def _is_on_antimeridian(geom):
+        return geom.geom_type == "MultiPolygon" and len(geom.geoms) > 1
+
+    epsgs = np.zeros(len(df), dtype=int)
+
+    # do the antimeridian frames first
+    am_idxs = df.geometry.map(_is_on_antimeridian).values
+    epsgs[am_idxs] = df[am_idxs].geometry.map(antimeridian_epsg)
+
+    # everything else
+    # get the x, y (lon, lat) coords of all other rows
+    other_coords = np.array(
+        df[~am_idxs].geometry.map(lambda g: tuple(g.centroid.coords)[0]).tolist()
+    )
+    xs, ys = other_coords.T
+    ys_full_size = np.ones(len(epsgs)) * np.nan
+    ys_full_size[~am_idxs] = ys
+
+    idxs = np.logical_and.reduce((~am_idxs, ys_full_size > 84))
+    epsgs[idxs] = 3413
+
+    idxs = np.logical_and.reduce((~am_idxs, ys_full_size < -60))
+    epsgs[idxs] = 3031
+
+    utm_idxs = np.logical_and(ys < 84, ys > -60)
```

Comment:
I've made them constants, and set it to +75 for now (I still think it would be nice to not have the very top of alaska as a different projection than the rest)